src/game/features/players/toxic/CagePlayerLarge.cpp

```diff
@@ -12,9 +12,9 @@ namespace YimMenu::Features
 
 		virtual void OnCall(Player player) override
 		{
-			auto coords = player.GetPed().GetPosition();
-			coords.z -= 1.0f;
-			SpawnObject(0x99C0CFCF, coords);
+			Vector3 coords = ENTITY::GET_ENTITY_COORDS(player.GetPed().GetHandle(), true, true);
```

Comment:
This shouldn't have been changed back to this. Revert it to this implementation:
```cpp
auto coords = player.GetPed().GetPosition();
coords.z -= 1.0f;
SpawnObject(0x99C0CFCF, coords, 0, 0, 0, true, true, false, false, true);
```